python/pylibcudf/pylibcudf/column.pyx

```diff
@@ -328,12 +336,35 @@ cdef class Column:
         self._children = children
         self._num_children = len(children)
 
+    def to_arrow(
+        self,
+        metadata: ColumnMetadata | str | None = None
+    ) -> ArrowLike:
+        """Create a PyArrow array from a pylibcudf column.
+
+        Parameters
+        ----------
+        metadata : list
+            The metadata to attach to the columns of the table.
+
+        Returns
+        -------
+        pyarrow.Array
+        """
+        if pa_err is not None:
+            raise RuntimeError(
+                "pyarrow was not found on your system. Please "
+                "pip install pylibcudf with the [pyarrow] extra for a "
+                "compatible pyarrow version."
+            ) from pa_err
+        return pa.array(_ObjectWithArrowMetadata(self, metadata))
```

Comment:
You should just be able to use `pa.array(self)` if `metadata is None`. I'm also is there a pressing use case for exposing `metadata`?

Reply:
I left the metadata arg as-is, I don't think it's likely to impact users.